build/yaml/crd/nsx.vmware.com_nsxserviceaccounts.yaml

```diff
@@ -35,6 +35,10 @@ spec:
           spec:
             description: NSXServiceAccountSpec defines the desired state of NSXServiceAccount
             properties:
+              enableCertRotation:
+                description: EnableCertRotation enables cert rotation feature in this
```

Comment:
do you mean this will go into nsx-keeper-413, is nsx code merged into the branch?

Reply:
Yes. Since NSXT 4.1.3 and 4.2.0, CCP cert can be updated.
Then we can create a new NSXServiceAccount with enableCertRotation=true to enable the cert rotation feature.

Reply:
The addon-controller can set enableCertRotation based on deployed interworking version. It is TBD now.